packages/monosize/src/utils/collectLocalReport.test.mts

```diff
@@ -65,7 +65,7 @@ describe('collectLocalReport', () => {
     await fs.promises.writeFile(reportAPath, JSON.stringify(reportA));
     await fs.promises.writeFile(reportBPath, JSON.stringify(reportB));
 
-    expect(await collectLocalReport({ root: rootDir })).toMatchInlineSnapshot(`
+    expect(await collectLocalReport({ root: rootDir, reportFilesGlob: undefined })).toMatchInlineSnapshot(`
```

Comment:
added explicit `undefined`  to properly test if that args is not specified from CLI